charts/posthog/values.yaml

```diff
@@ -338,6 +338,8 @@ ingress:
   annotations: {}
   # -- TLS secret to be used by the ingress.
   secretName:
+  # -- Troubleshooting tool to swich to http direct access to the IP. This overrides hostname, nginx.redirectToTLS, gcp.forceHttps, letsencrypt and web.secureCookies to false. Don't use in Production.
```

Comment:
In general from what I've seen helm charts try to avoid having a lot of "business logic" inside the charts and values, instead being a straight-forward mapping from values to a kubernetes manifest.

I like that. Having values which have so large of an impact is a trap, because while it looks appealing from the outside, it:
1. Will cause surprises for our users (e.g. some of their values will be overwritten)
2. It will cause bugs as we update ingestion in general - e.g. we won't remember to consider this case in particular